pkg/plugin/interface.go

```diff
@@ -41,27 +40,21 @@ type Interface interface {
 	// ExpectedResults is an array of Result objects that a plugin should
 	// expect to submit.
 	ExpectedResults(nodes []v1.Node) []ExpectedResult
+	// FillTemplate fills the driver's internal template so it can be presented to the user for `sonobuoy gen`
```

Comment:
nit: referencing the actual application command will probably become stale in not much time